src/components/extras/embeddable-page.vue

```diff
@@ -0,0 +1,94 @@
+<!--
+@Author: Adam Oberg <Aadm
+@Date:   2020-05-06 T:22:00:00-22:30:00
+@Email:  adam.oberg@oregonstate.edu
+@Last modified by:   Adam
+@Last modified time: 2020-05-06 T:22:00:00-22:30:00
+-->
+
+<template>
```

Comment:
Looks like you copied the kiosk page layout. Thats not quite right for the energy dashboard. Look at some of the other components for examples. It shouldn't have el-container, header etc.